doc/xds-test-descriptions.md

```diff
@@ -10,7 +10,12 @@ The code for the xDS test server can be found at:
 Server should accept these arguments:
 
 *   --port=PORT
-    *   The port the server will run on.
+    *   The port the test server will run on.
+*   --maintenance_port=PORT
+    *   The port for the maintenance server running admin, health and reflection services.
```

Comment:
admin isn't a service so maybe "admin services", and we don't care about reflection for the tests right? We care about this port for health, CSDS, and channelz right?

Reply:
My description was based on the following code for the maintenance server:
```
          NettyServerBuilder.forPort(maintenancePort)
              .addService(new XdsUpdateHealthServiceImpl(health))
              .addService(health.getHealthService())
              .addService(ProtoReflectionService.newInstance())
              .addServices(AdminInterface.getStandardServices())
```
Whether we care about some service or not is subjective. For example, the `XdsUpdateHealthServiceImpl` service is important for the test driver since it is part of the tests but we don't specifically mention it (unless it is part of the "health" services). In any case I'll update the wording to reflect your suggestion.